src/call_builder.ts

```diff
@@ -190,7 +224,7 @@ export class CallBuilder {
     }
 
     // Temp fix for: https://github.com/stellar/js-stellar-sdk/issues/15
-    url.setQuery('c', Math.random());
+    url.setQuery('c', Math.random() as any);  // TODO: Type conflict, please review.
```

Comment:
Seems like this should cast as string? If by some chance this gets tested to return a number, which I think is unlikely (but anything can happen), I think the test should be fixed.